src/CMakeLists.txt

```diff
@@ -72,6 +72,10 @@ if(BUILD_P11)
   add_definitions(-DBUILD_P11)
 endif(BUILD_P11)
 
+if(ACCEPT_LOWER_VERSION)
+  add_definitions(-DACCEPT_LOWER_VERSION)
```

Comment:
I wonder if we could define `ACCEPT_LOWER_VERSION` as `true` or `false` here so we don't need the additional macro definition `AKLITE_ACCEPT_LOWER_VER_DEFAULT`?

Reply:
We have to use 1 or 0 as values then, I suppose.

Reply:
IMO, we would need a `#ifndef` block anyway, to avoid making the macro definition obligatory when building. And, although it would work for the current implementation, we may want to implement the downgrade control in a different way, so using this more abstract approach seems to be better.